### PR TITLE
Update best-practices-for-configuring-eop.md

### PR DESCRIPTION
**Pull request by andypunt**
Moved out expired items and move some to Recommended settings list.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/best-practices-for-configuring-eop.md [hunk 1]
```diff
@@ -14,44 +14,51 @@ description: "Follow these best-practice recommendations for Exchange Online Pro
...
 
-## SPF record customization to help prevent spoofing
+## Recommended settings
 
-When you set up EOP, you added an SPF (sender policy framework) record for EOP to your DNS records. The SPF record helps prevent spoofing. For more information about how an SPF record prevents spoofing and how you can add your on-premises IP addresses to the SPF record, see [Set up SPF in Office 365 to help prevent spoofing](set-up-spf-in-office-365-to-help-prevent-spoofing.md).
+We empower security administrators to customize their security settings that satistfy the needs of their environments. Although, as a general rule, there are two security levels in EOP and Office 365 ATP that we recommend: Standard and Strict. These settings are listed in the [Recommended settings for EOP and Office 365 ATP security](recommended-settings-for-eop-and-office365-atp.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/best-practices-for-configuring-eop.md]:
```suggestion
We empower security administrators to customize their security settings that satistfy the needs of their environments. Although, as a general rule, there are two security levels in EOP and Office 365 ATP that we recommend; Standard and Strict. These settings are listed in the [Recommended settings for EOP and Office 365 ATP security](recommended-settings-for-eop-and-office365-atp.md). 
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by andypunt** [comment]
I am not sure how to accepted all changes and it seems the PR still has conflicts. Can you help resolve?

**Comment by andypunt** [comment]
Let me take a look at it when I get out of this meeting.

Thanks,
Tracy

From: Andy Puntahachart <andypunt@microsoft.com>
Sent: Monday, November 25, 2019 11:34 AM
To: Daniel Simpson <Daniel.Simpson@microsoft.com>; Tracy Paddock <tracyp@microsoft.com>; Chris Davis (EXCHANGE) <Chris.Davis@microsoft.com>
Cc: Author <author@noreply.github.com>; MicrosoftDocs/microsoft-365-docs <reply@reply.github.com>; MicrosoftDocs/microsoft-365-docs <microsoft-365-docs@noreply.github.com>
Subject: RE: [MicrosoftDocs/microsoft-365-docs] Update best-practices-for-configuring-eop.md (#778)

I am not sure how to accepted all changes and it seems the PR still has conflicts. Can you help resolve?

**Review by MSFTTracyP — APPROVED** [review]
Good idea

**Review by MSFTTracyP — APPROVED** [review]
Merge

**Pull request merged**